src/ui/public/styles/base.less

```diff
@@ -577,30 +565,32 @@ fieldset {
 }
 
 .kuiSideBarSection {
-  margin-bottom: 6px;
+  margin-bottom: 10px;
+  background-color: @globalColorWhite;
+  padding: 10px;
+  border-radius: 4px;
+}
+
+.kuiSideBarSection .kuiSideBarSection .kuiSideBarSection{
```

Comment:
I'd rather do this in a separate refactor. Currently those three tabs are all using different markup and I'm faking them all to look the same with separate css. This whole bit needs a lot of work and because of the way these templates sub nested each other I didn't have enough angular to better specify the classes (if I recall correctly).